soh/soh/Enhancements/cosmetics/CosmeticsEditor.cpp

```diff
@@ -1663,7 +1663,7 @@ void DrawCosmeticRow(CosmeticOption& cosmeticOption) {
         LUS::Context::GetInstance()->GetWindow()->GetGui()->SaveConsoleVariablesOnNextTick();
     }
     ImGui::SameLine();
-    ImGui::Text(cosmeticOption.label.c_str());
+    ImGui::Text("%s", cosmeticOption.label.c_str());
```

Comment:
Shame about this being a warning. It's just kinda making it more verbose for no real gain. Oh well.